deeppavlov/skills/go_bot/network.py

```diff
@@ -184,10 +210,9 @@ def _add_placeholders(self):
 
     def _build_body(self):
         # input projection
-        _units = tf.nn.dropout(self._features, self._dropout)
-        _units = tf.layers.dense(_units, self.dense_size,
-                                 kernel_initializer=xav(), name='units')
-
+        _units = tf.layers.dense(self._features, self.dense_size,
```

Comment:
May be some activation function here?

Reply:
Sounds good, but not in the pull-request =)